examples/acoustic/acoustic_example.py

```diff
@@ -42,7 +42,7 @@ def run(dimensions=(150, 150, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
     # Smooth velocity
     initial_vp = smooth10(true_vp, dimensions)
 
-    dm = true_vp**-2 - initial_vp**-2
```

Comment:
If true_vp isn't changed in place by smooth10, you can compute the value before (once and for all), instead of doing it on very grid point